### ISSUE TITLE
Which version of Pillow-SIMD works with Pilbox 1.3.4?

### ISSUE THREAD

**Issue by xtagon** [agschwender/pilbox#71]
Is there a recommended version of Pillow-SIMD that works with Pilbox 1.3.4? They don't have a tag for 5.2.0 - I tried 5.1.1.post0 (the latest) but it caused errors:

```
ImportError: The _imaging extension was built for another version of Pillow or PIL: 
```

**Comment by agschwender** [comment]
Looks like it doesn't support 5.2.0 yet. I'd have to make a new version that used an earlier Pillow version.

**Comment by agschwender** [comment]
It looks like they released their 5.2.0 version: https://pypi.org/project/Pillow-SIMD/5.2.0.post0/

**Comment by xtagon** [comment]
Cool, thank you for the tip!
